### PR TITLE
docs(readme): Update readme with usage instructions for alpha.

### PR DESCRIPTION
**Pull request by jtk54**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,9 @@ auth:
     keyPath: ~/.spin/keyfile
     certPath: ~/.spin/certfile
 ```
+
+# Usage
+
+Download the `spin` binary from a [release](https://github.com/spinnaker/spin/releases) and
```
Comment (lwander) [inline README.md]: How is the download done? Can we provide an easy curl command?

Comment (jtk54) [reply README.md]: Is there a latest release url for github?

Comment (jtk54) [reply README.md]: I guess we could set a `$VERSION` and then give a curl command.

**Comment by jtk54** [comment]
How about this?

**Comment by lwander** [comment]
Do we have to crosscompile for darwin or linux? Or just say this is linux only?

**Comment by jtk54** [comment]
Linux only for now in the pre-release, I'll mark it there. If we want, I can figure out how to make a cross-compatible release for a proper alpha.

**Pull request merged**